### DISCUSSION TITLE
Process: New KIT

### DISCUSSION THREAD

**Discussion by efficient0482** [eclipse-tractusx/sig-release#7] [Ideas]
If someone wants to contribute by developing a new KIT we need a Process. 

Prerequisites:
 - Github repository

Process: 
1. KIT Team raises an Issue using the "request new KIT" - Issue template (introduced by  #5 )
2. Community reviews the request and decides if this request should be accepted
3.  upon acceptance, add the repository to the tractus-x project.

I look forward to your thoughts and remarks

**Comment by FaGru3n** [comment]
@giterrific @danielmiehle @Siegfriedk 
 
Hi,
general question for me should we intergrate also the requesting new kit´s in sig-release repo or should we create a separate repo for kit handling? proposal like "sig-kits" as stated in @LuisRickert PR #5 

https://github.com/eclipse-tractusx/sig-release/pull/6#issuecomment-1725460552
 
 first it will a bit harder for User of sig-release repo to understand the issues for release management / Feature Bug Handling / and Kit-Request...

**Reply by Siegfriedk** [reply]
I would say 'depends on the amount'? Most documentation / infos regarding kits are currently in our eclipse-tractus-x page itself and not in any sig.

**Reply by FaGru3n** [reply]
already discussed in the PR and think we can close this discussion here at this point...
https://github.com/eclipse-tractusx/sig-release/pull/5#issuecomment-1736955294
